docs/api/toast.md

```diff
@@ -64,6 +64,14 @@ import ButtonsPlayground from '@site/static/usage/v7/toast/buttons/index.md';
 
 Toasts can be positioned at the top, bottom or middle of the viewport. The position can be passed upon creation. The possible values are `top`, `bottom` and `middle`. If the position is not specified, the toast will be displayed at the bottom of the viewport.
 
+### Handling Navigation Elements
```

Comment:
Perhaps this should say something like "Relative to navigation elements"? I'm not sure.

Reply:
Changed to "Relative Positioning" to be a little snappier, open to other ideas though 👀